src/home/fetch-github/fetch-and-display-previews.ts

```diff
@@ -76,31 +77,37 @@ function filterIssuesByOrganization(issues: GitHubIssue[]): GitHubIssue[] {
 export async function displayGitHubIssues({
   sorting,
   options = { ordering: "normal" },
+  filterAvailableIssues = true,
   skipAnimation = false,
 }: {
   sorting?: Sorting;
   options?: { ordering: string };
+  filterAvailableIssues?: boolean;
```

Comment:
akshually I disagree with this being passed as an argument cause if u forget to properly pass it will do wrong i.e: display on search. it'd be best to have a global as we do with proposals!